src/eko/evolution_operator/__init__.py

```diff
@@ -32,25 +32,27 @@
 )
 
 
-@nb.njit("c16(c16[:,:],string)")
-def select_singlet_element(ker, mode):
+@nb.njit("c16(c16[:,:],u2,u2)")
+def select_singlet_element(ker, mode0, mode1):
     """
     Select element of the singlet matrix
 
     Parameters
     ----------
-        mode : str
-            sector element
         ker : numpy.ndarray
             singlet integration kernel
-
+        mode0 : int
+            first sector element
```

Comment:
```suggestion
            pid for first sector element
```
same for `mode1`, to be consistent

Reply:
Maybe here I'd write `id` more generically: the `pid` is usually taken to be one of those listed in PDG, but here we're making up `id`s ourselves (like for singlet), so no need to generate more confusion, let's just drop one letter.